src/com/lftechnology/java/training/niraj/smallestmultiple/Multiples.java

```diff
@@ -0,0 +1,55 @@
+package com.lftechnology.java.training.niraj.smallestmultiple;
+
+/**
+ * Helper class that consists of helper functions related to multiples
+ * 
+ * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+ */
+public class Multiples {
+
+	private int minNum;
+	private int maxNum;
+
+	public Multiples(int minNum, int maxNum) {
+		this.minNum = minNum;
+		this.maxNum = maxNum;
+	}
+
+	/**
+	 * Gets the smallest multiple that divisible evenly
+	 * 
+	 * @return {@link Integer} smallest number evenly divisible by numbers in
+	 *         certain range
+	 * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+	 */
+	public int smallestMultiple() {
+		int smallestMultiple = 0;
+		int num = maxNum;
+		for (;;) {
```

Comment:
It would better to use following logic to get smallest multiple number looping infinite loop and checking every value with each number is divided or not: 
```
int smallerMultipleNum = from;
int sumNumber; 
for (int i = from; i <= to; i++) {
	sumNumber = smallerMultipleNum;
	while (sumNumber % i != 0) {
		sumNumber += smallerMultipleNum;
	}	
	smallerMultipleNum = sumNumber;
}
return smallerMultipleNum;
```
If you have to get smallest number that is divisible by all of the numbers from 1 to 20, above code will find smallest number in 74 loop. 
But in your condition it will find smallest number in 232792540 * 20 loop.  
So it will increase perforation of your application.